File: papi/lista1/exercicio1.py
while (True):
    try:
        A = input("Por favor insira o valor A: ")
        if A == "0":
            A = 0
            break
        A = int(A)
        if A != 0:
            break
    except:
        print("Por favor digite um número")

while (True):
    try:
        B = input("Por favor insira o valor B: ")
        if B == "0":
            B = int(B)
            break
        B = int(B)
        if B != 0:
            break
    except:
        print("Por favor digite um número")

while (True):
    try:
        C = input("Por favor insira o valor C: ")
        if C == "0":
            C = int(C)
            break
        C = int(C)
        if C != 0:
            break
    except:
        print("Por favor digite um número")


soma_AB = A + B

print("A soma de A e B é:", soma_AB)

if soma_AB > C:
    print("A soma entre A e B é maior que C")
elif soma_AB == C:
    print("A soma entre A e B é igual a C")    
else:
    print("A soma entre A e B é menor que C")

File: papi/lista1/exercicio1.kt
package lista1

fun main() {
    var numero1: Float
    var numero2: Float
    var numero3: Float

    while (true) {
        try {
            print("Digite o número 1: ")
            var input = readLine()
            numero1 = input!!.toFloat()
            break
        } catch(e: Exception) {
            println("Informe um valor numérico, usando o ponto como separador decimal")
        }
    }

    while (true) {
        try {
            print("Digite o número 2: ")
            var input = readLine()
            numero2 = input!!.toFloat()
            break
        } catch(e: Exception) {
            println("Informe um valor numérico, usando o ponto como separador decimal")
        }
    }

    while (true) {
        try {
            print("Digite o número 3: ")
            var input = readLine()
            numero3 = input!!.toFloat()
            break
        } catch(e: Exception) {
            println("Informe um valor numérico, usando o ponto como separador decimal")
        }
    }
    
    val soma = numero1 + numero2
    println("A soma ente o número 1 e o número 2 é $soma")

    if (soma < numero3) {
        println("A soma entre o número 1 e o número 2 é menor que o número 3")
    } else if (soma == numero3) {
        println("A soma entre o número 1 e o número 2 é igual ao número 3")
    } else {
        println("A soma entre o número 1 e o número 2 é maior que o número 3")
    }
}